management.py

```diff
@@ -0,0 +1,19 @@
+from model import Model
+
+INITIAL_MONEY = 1000
+SPENDING_PROB = 0.5 
+SPENDING_AMOUNT = 100
+NUM_PEOPLE = 10
+SALARY = 1000
+NUM_STEPS = 10
```

Comment:
A few suggestions here. I recommend moving these values as default values to the constructor of Model(). See below.